validator/modules/onnx/train_svm.py

```
@@ -0,0 +1,160 @@
+import pandas as pd
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)